functorch/_src/aot_autograd.py

```diff
@@ -410,7 +414,6 @@ def disable_autocast_manager():
     finally:
         del guard
 
-
```

Comment:
So much line noise wryyyy

Reply:
this diff has lived a storied life, ill clean it up

Reply:
I think it might be the linter lol.